doc/news/_preparation_next_release.md

```diff
@@ -210,8 +210,8 @@ Thanks to Daniel Bugl.
 
 ### Core
 
-- The KeySet has a new flag `KS_FLAG_REBUILD_OPMPHM`, this flag will be used by the hybrid search
-  that combines binary search and [OPMPHM](https://master.libelektra.org/doc/dev/data-structures.md#order-preserving-minimal-perfect-hash-map-aka-opmphm) *(Kurt Micheli)*
+- The KeySet has a new flag `KS_FLAG_NAME_CHANGE`, this flag will be used by the hybrid search
```

Comment:
What does the flag do? And isn't it a rename of an existing flag?

Please move text to a section "Proposal", it is not yet part of Core.

Reply:
A keyset is marked with this flag when a key name has changed. It is a existing one but not jet used.